packages/types/src/Metadata/v5/toV6.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import MetadataV5 from './Metadata';
+import MetadataV6 from '../v6';
+
+/**
+ * Convert from MetadataV5 to MetadataV6
+ * See https://github.com/polkadot-js/api/issues/1043 for details
+ */
+export default function toV6 (metadataV5: MetadataV5): MetadataV6 {
+  return new MetadataV6({
+    modules: metadataV5.modules.map((modul) => {
+      return {
+        name: modul.name,
+        prefix: modul.prefix,
+        storage: modul.storage,
+        calls: modul.calls,
+        events: modul.events,
+        constants: []
```

Comment:
Here it should be None, i.e. `new Option.with(...)(undefined)`

Reply:
how about `new Vector(ModuleConstantMetadata, [])` ? since constants is not an optional

Reply:
Ah! My bad, i thought is was optional like the 3 above. Then let's keep it like this.